lib/icingadb/icingadb-objects.cpp

```diff
@@ -290,6 +341,133 @@ void IcingaDB::UpdateAllConfigObjects()
 			}
 		}
 
+		std::map<String, std::map<String, String>> ourContent;
+
+		for (auto& source : ourContentRaw) {
+			upqObjectType.Enqueue([&]() {
+				auto& dest (ourContent[source.first]);
+
+				for (auto& hMSet : source.second) {
+					auto stop (hMSet.size());
+
+					if (stop % 2u) {
+						--stop;
+					}
+
+					for (decltype(stop) i = 0; i < stop; i += 2u) {
```

Comment:
`i < hMSet.size() - 1` would be simpler, i.e. iterate as long as accessing both `hMSet[i]` and `hMSet[i+1]` is safe.